### PR TITLE
Appsflyer doc hotfix

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## What do these changes do?

> Added missing dashboard settings in the AppsFlyer doc.
> Some minor doc cleanup and standardization.

## Type of documentation

- [ ] New documentation
- [ ] Documentation update
- [x] Hotfix

### CODE REVIEW

File: docs/destinations/streaming-destinations/appsflyer.mdx [hunk 1]
```diff
@@ -3,94 +3,63 @@ title: "AppsFlyer"
...
   - **Android App ID**: This the application ID used in your `app/build.gradle` file.
-
   - **Apple App ID**: This is the iTunes Application ID and it is mandatory for the iOS applications.
 
 <div class="infoBlock">
-
-If you are connecting a source other than <strong>Android</strong> or the <strong>Apple</strong> (iOS, watchOS, iPadOS, tvOS), you need to add a user transformation that adds the OS information required in the events.
-
-Refer to the <a href="#connecting-appsflyer-to-non-androidios-sources">Connecting AppsFlyer to non-Android/iOS sources</a> section below for more information.
+If you are connecting a source other than Android or Apple (iOS, watchOS, iPadOS, tvOS), you need to add a user transformation that adds the OS information required in the events. Refer to the <a href="#connecting-appsflyer-to-non-androidios-sources">Connecting AppsFlyer to non-Android/iOS sources</a> section below for more information.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/appsflyer.mdx:44]:
```suggestion
To connect a source other than Android or Apple (iOS, watchOS, iPadOS, tvOS), you need to add a user transformation that adds the OS information required in the events. Refer to the <a href="#connecting-appsflyer-to-non-androidios-sources">Connecting AppsFlyer to non-Android/iOS sources</a> section below for more information.
```

File: docs/destinations/streaming-destinations/appsflyer.mdx [hunk 2]
```diff
@@ -3,94 +3,63 @@ title: "AppsFlyer"
...
 
-- **Sharing Filter**: By default, the value for this setting is set to `all`. You can use this setting to meet any regulatory requirements like GDPR and CCPA, complying with user opt-out mechanisms, and for any other business use-case.
-
-<div class="infoBlock">
-
-For more information on this field, refer to the <a href="https://support.appsflyer.com/hc/en-us/articles/207034486-Server-to-server-events-API-for-mobile-S2S-mobile-#sharing_filter-16">AppsFlyer Help Center</a> page.
-</div>
-
-- **Client-side Event Filtering**: This setting is applicable only if you are sending events to AppsFlyer via device mode. 
+- **Client-side Event Filtering**: This setting is applicable only if you are sending events to AppsFlyer via device mode. It lets you specify which events should be blocked or allowed to flow through to AppsFlyer. Refer to the <Link to="/sources/event-streams/sdks/event-filtering/">Client-side Event Filtering</Link> guide for more information on this setting.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/appsflyer.mdx:50]:
```suggestion
- **Client-side Event Filtering**: This setting is applicable **only if** you are sending events to AppsFlyer via device mode. It lets you specify which events should be blocked or allowed to flow through to AppsFlyer. Refer to the <Link to="/sources/event-streams/sdks/event-filtering/">Client-side Event Filtering</Link> guide for more information on this setting.
```

File: docs/destinations/streaming-destinations/appsflyer.mdx [hunk 3]
```diff
@@ -724,6 +671,10 @@ RudderStack sets `email` from the event traits to AppsFlyer using the native SDK
 
 ### Deleting a user
 
+<div class="warningBlock">
+To delete a user in AppsFlyer, you must configure the <strong>Status Callback URLs</strong> and <strong>API Token</strong> settings in the RudderStack dashboard.
+</div>
+
```
Comment (utsabc) [inline docs/destinations/streaming-destinations/appsflyer.mdx:677]: Can we also talk about api-token

File: docs/destinations/streaming-destinations/appsflyer.mdx [hunk 4]
```diff
@@ -3,94 +3,63 @@ title: "AppsFlyer"
...
-- **Client-side Event Filtering**: This setting is applicable only if you are sending events to AppsFlyer via device mode. 
+- **Client-side Event Filtering**: This setting is applicable **only if** you are sending events to AppsFlyer via device mode. It lets you specify which events should be blocked or allowed to flow through to AppsFlyer. Refer to the <Link to="/sources/event-streams/sdks/event-filtering/">Client-side Event Filtering</Link> guide for more information on this setting.
 
-<div class="infoBlock">
-  
-Refer to the <Link to="/sources/event-streams/sdks/event-filtering/">Client-side Event Filtering</Link> guide for more information on this setting.
-</div>
+- **Status Callback URLs**: Specify the callback URLs to be used for user deletion requests. You can provide multiple callback URLs by separating them by a comma.
 
-- **Use native SDK to send events**: By default, this setting is enabled. To send events via cloud mode or device mode, you can toggle this setting on or off.
```
[the remark this answers is not in the document]
Comment (ishachopra0401) [reply docs/destinations/streaming-destinations/appsflyer.mdx:82]: added

File: docs/destinations/streaming-destinations/appsflyer.mdx (hunk 3 above)
Comment (ishachopra0401) [reply docs/destinations/streaming-destinations/appsflyer.mdx:680]: @utsabc API token is already mentioned as being mandatory in the warning block. Can you please clarify what else should be modified?

Comment (utsabc) [reply docs/destinations/streaming-destinations/appsflyer.mdx:680]: Oh oops I missed it nps

**Review by utsabc — APPROVED** [review]

**Pull request merged**